src/chains/filecoin/types/src/api.d.ts

```diff
@@ -12,6 +12,9 @@ import { SerializedFileRef } from "./things/file-ref";
 import { SerializedMinerPower } from "./things/miner-power";
 import { SerializedMinerInfo } from "./things/miner-info";
 import { SerializedVersion } from "./things/version";
+import { SerializedMessage } from "./things/message";
+import { SerializedMessageSendSpec } from "./things/message-send-spec";
+import { SerializedSignedMessage } from "./things/signed-message";
 export default class FilecoinApi implements types.Api {
   #private;
```

Comment:
does this do anything?

Reply:
Not that I know of, but everything under `src/chains/filecoin/types/src` is autogenerated by `ttsc` with the `npm run tsc.filecoin.declarations` command